examples/advanced_circuits_algorithms/Deutsch_Jozsa_Algorithm/Deutsch Jozsa Algorithm.ipynb

```diff
@@ -0,0 +1,1047 @@
+{
```

Comment:
<p>I would change 'We create a list of different types of gates you can apply, the X gate a single gate and CNOT and CCNOT require multiple gates.' to 'We create a list of different types of gates you can apply, the X gate, which is a single qubit gate, and CNOT and CCNOT gates, which are multiple qubit gates.'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/177/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
Updated in most recent PR